src/MediaRatings.java

```diff
@@ -0,0 +1,62 @@
+/**
+ * MediaRatings secondary methods.
+ *
+ * @param <K>
+ *            the type of the key entries
+ * @param <V>
+ *            the type of the value entries
+ */
+public interface MediaRatings<K, V> extends MediaRatingsKernel<K, V> {
```

Comment:
I don't know that you need generics here unless you plan to let the user specify their rating types. You could honestly hardcode the types you want and leave the generics out altogether.